tests/e2e/local/minikube/install_prereqs_centos.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+# Check if yum is installed
+if ! yum --help > /dev/null; then
+    echo "yum not installed. Please install it manually and run this script again."
+    exit 1
+fi
+sudo yum --quite -y update
+
+# Check if rpm is installed
+if ! rpm --help > /dev/null; then
+    echo "rpm not installed. Please install it and run this script again."
+    exit 1
+fi
+
+#Install Curl
+echo "Checking and Installing Curl as required"
+if ! curl --help > /dev/null; then
+    sudo yum -y install curl
+    if ! curl --help > /dev/null; then
+      echo "curl could not be installed. Please install it manually and run this script again."
+      exit 1
+    fi
+fi
+
+#Install Kvm2
+echo "Installing KVM2 as required"
+sudo modprobe kvm > /dev/null 2>&1
+sudo yum install -y qemu-kvm qemu-img virt-manager libvirt libvirt-python libvirt-client libguestfs-tools virt-install virt-viewer bridge-utils
+sudo systemctl start libvirtd
+sudo systemctl enable libvirtd
```

Comment:
I believe the step at line 31 is redundant, although I could be mistaken - centos and libvirt change often.  The behavior I am most used to is libvirt enabled by default after an installation.

Reply:
Usually I use `systemctl enable libvirtd` to make sure libvirtd is enabled.

I find we do not explictly enable libvirtd in install_prereqs_debian.sh, so I will remove this line to make it consistent.